netutils/nanopb/Kconfig

```diff
@@ -0,0 +1,41 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+config NETUTILS_NANOPB
+	tristate "Nanopb Protocol Buffers"
+	default n
+	---help---
+		Nanopb is a small code-size Protocol Buffers implementation in ansi C.
+		It is especially suitable for use in microcontrollers, but fits any
+		memory restricted system.
+		Important: the example option requires Python to be installed
+		and also the following packages:
```

Comment:
"and also the following packages:" I think the packages name are missing

Reply:
Thanks! This line is not needed anymore since the download tarball contains the executables, no Python is necessary.